packages/aws-cdk-lib/aws-ec2/lib/vpc-endpoint.ts

```diff
@@ -316,9 +319,11 @@ export class InterfaceVpcEndpointAwsService implements IInterfaceVpcEndpointServ
   public static readonly CLOUDFORMATION = new InterfaceVpcEndpointAwsService('cloudformation');
   public static readonly CLOUDHSM = new InterfaceVpcEndpointAwsService('cloudhsmv2');
   public static readonly CLOUDTRAIL = new InterfaceVpcEndpointAwsService('cloudtrail');
+  /** @deprecated Use InterfaceVpcEndpointAwsService.Q_DEVELOPER_CODE_WHISPERER instead.*/
```

Comment:
CodeWhisperer has beed rebranded to Amazon Q Developer.

Ref:
* https://docs.aws.amazon.com/ja_jp/codewhisperer/latest/userguide/whisper-legacy.html
* https://aws.amazon.com/about-aws/whats-new/2024/04/amazon-q-developer-generally-available/?nc1=h_ls